pkg/controller/utils/utils_test.go

```diff
@@ -0,0 +1,57 @@
+package utils
+
+import (
+	"testing"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMergeJsons(t *testing.T) {
+	tests := []struct {
+		name        string
+		jsonStr1    string
+		jsonStr2    string
+		expectedStr string
+		expectedErr bool
+	}{
+		{
+			name:        "Merge pull secrets 01",
+			jsonStr1:    `{"auths": {"registry.svc.ci.openshift.org": {"auth": "dXNljlfjldsfSDD"}}}`,
+			jsonStr2:    `{"auths":{"cloud.openshift.com":{"auth":"b34xVjWERckjfUyV1pMQTc=","email":"lmohanty@redhat.com"},"quay.io":{"auth":"b3BlbnNoVkc=","email":"lmohanty@redhat.com"},"registry.connect.redhat.com":{"auth":"NjQ4ODeDZ3d1pN","email":"lmohanty@redhat.com"},"registry.redhat.io":{"auth":"NjQ4ODX1pN","email":"lmohanty@redhat.com"}}}`,
```

Comment:
I suggest that you use dummy email addresses like "user@email.com" instead of your own email address.